From b1a3217c5cd61e5e3a2de5ed5af8f77ea2aa792a Mon Sep 17 00:00:00 2001
From: "Corrado Lanera (xps)" <corrado.lanera@gmail.com>
Date: Wed, 7 Feb 2018 15:51:43 +0100
Subject: [PATCH] Fix bug pipe docs

---
 R/utils-pipe.R | 2 +-
 man/pipe.Rd    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/R/utils-pipe.R b/R/utils-pipe.R
index fb8c818..c76331d 100644
--- a/R/utils-pipe.R
+++ b/R/utils-pipe.R
@@ -1,6 +1,6 @@
 #' Pipe operator
 #'
-#' See \code{magrittr::\link[magrittr]{\%>\%}} for details.
+#' See \code{magrittr::\link[magrittr]{pipe}} for details.
 #'
 #' @name %>%
 #' @rdname pipe
diff --git a/man/pipe.Rd b/man/pipe.Rd
index b7daf6a..5348f4b 100644
--- a/man/pipe.Rd
+++ b/man/pipe.Rd
@@ -7,6 +7,6 @@
 lhs \%>\% rhs
 }
 \description{
-See \code{magrittr::\link[magrittr]{\%>\%}} for details.
+See \code{magrittr::\link[magrittr]{pipe}} for details.
 }
 \keyword{internal}
-- 
2.15.1.windows.2

